pipeline_steps/influx.groovy

```diff
@@ -0,0 +1,67 @@
+def setup(){
+  instance_name = "INFLUX"
+  pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
+  common.override_inventory()
+  try{
+    common.conditionalStage(
+      stage_name: "Influx",
+      stage:{
+        withCredentials([
+          file(
+            credentialsId: 'id_rsa_cloud10_jenkins_file',
+            variable: 'jenkins_ssh_privkey'
+          ),
+        ]){
+          dir('/tmp/'){
+            unstash "pubcloud_inventory"
```

Comment:
not sure what you meant here. I thought this line was staging the pubcloud_inventory [1]
I thought the playbooks were started from the jenkins node, therefore I had to give the inventory to connect to the influxdb node, and run my own playbooks.

[1]: https://github.com/rcbops/rpc-gating/blob/master/pipeline_steps/pubcloud.groovy#L33  and is called somewhere in the stack.